crates/outbound-http/src/host_impl.rs

```diff
@@ -28,11 +27,13 @@ impl OutboundHttp {
     /// If `None` is passed, the guest module is not allowed to send the request.
     fn is_allowed(&mut self, url: &str) -> Result<bool, HttpError> {
         if url.starts_with('/') {
-            return Ok(self.allowed_hosts.allows_relative_url());
+            return Ok(self.allowed_hosts.allows_relative_url("http"));
```

Comment:
Is this going to fail if "self" is using https?

Reply:
Yes, We should probably allow either.